core/src/main/scala/ammrunner/AmmoniteFetcher.scala

```diff
@@ -59,23 +59,23 @@ import scala.io.{BufferedSource, Codec}
         if (interpOnly)
           "ammonite-interp-api_" + versions.scalaVersion
         else
-          "ammonite-repl-api_" + versions.scalaVersion,
+          "ammonite-repl-api_" + twoThreeInterop(versions.scalaVersion),
```

Comment:
Looking at the artifacts, I believe this is the way to go right? Since `ammonite-repl-api` isn't actually published for Scala 3, but is 2.13 and just being used for the Scala 3 support?